qdrant-landing/content/documentation/cloud/pricing-payments.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Pricing & Payments
+weight: 65
+aliases:
+  - aws-marketplace
+  - gcp-marketplace
+  - azure-marketplace
+---
+
+# Pricing & Payments
+
+Qdrant database clusters in Qdrant Cloud are prices based on CPU, memory, and disk storage usage. To get a clearer idea for the pricing structure, based on the amounts of vectors you want to store, please use our [Pricing Calculator](https://cloud.qdrant.io/calculator).
+
+## Billing
+
+The costs for Qdrant Cloud database clusters can be paied either through a credit card or through the AWS, GCP, or Azure Marketplace.
```

Comment:
'Paid' instead of 'Paied'

But I think

'You can pay for your Qdrant Cloud database clusters either with a credit card or through an AWS, GCP, or Azure Marketplace subscription.

Might be better